JabMapTestData/dummydata-1.bib

```diff
@@ -0,0 +1,63 @@
+% Encoding: UTF-8
+@INPROCEEDINGS{aPaper,
+	author={T. {Kim} and S. {Park} and T. {Lee}},
+	booktitle={2017 IEEE International Symposium on Software Reliability Engineering Workshops (ISSREW)}, 
+	title={Applying software reliability engineering process to software development in Korea defense industry}, 
+	year={2017},
+	volume={},
+	number={},
+	pages={81-81},
+}
+
+@INBOOK{aBook,
+	author={D. {Galin}},
+	booktitle={Software Quality: Concepts and Practice}, 
+	title={Software Testing}, 
+	year={2018},
+	volume={},
+	number={},
+	pages={255-317},
+}
+
+@MindMapNode{1,
+	id = {1},
+	name = {a node},
+	bibentry = {aBook},
+	icons = {to_read},
+	x_pos = {2},
+	y_pos = {2},
+}
+
+@MindMapNode{2,
+	id = {2},
+	name = {test number 2},
+	bibEntry = {aPaper},
+	icons = {read,low_priority},
```

Comment:
snake_case is OK for me. Maybe we need to write an ADR on that. (camel case, snake case, separataion by spaces